test/unit/BondingManager.js

```diff
@@ -174,17 +178,22 @@ contract("BondingManager", accounts => {
                             })
                         }))
 
-                        bondingManager.TranscoderUpdate({transcoder: newTranscoder}).on("data", e => {
-                            assert.equal(e.returnValues.pendingRewardCut, 5, "should fire TranscoderUpdate event with provided rewardCut")
-                            assert.equal(e.returnValues.pendingFeeShare, 10, "should fire TranscoderUpdate event with provided feeShare")
-                            assert.equal(e.returnValues.registered, true, "should fire TranscoderUpdate event with registered set to true")
-                        })
-
                         const totalBonded = (await bondingManager.getTotalBonded()).toNumber()
 
                         // Caller bonds 6000 which is more than transcoder with least delegated stake
                         await bondingManager.bond(6000, newTranscoder, {from: newTranscoder})
-                        await bondingManager.transcoder(5, 10, {from: newTranscoder})
+                        const txRes = await bondingManager.transcoder(5, 10, {from: newTranscoder})
+                        truffleAssert.eventEmitted(
+                            txRes,
+                            "TranscoderUpdate",
+                            e => {
```

Comment:
Since the rest of this PR omits the brackets and `return`, let's do the same here for consistency?

Reply:
fixed in 35f5b36